python/xoscar/collective/tests/test_pygloo.py

```diff
@@ -0,0 +1,615 @@
+import multiprocessing as mp
+import os
+import platform
+import shutil
+import time
+
+import numpy as np
+
+from ...tests.core import require_linux, require_unix
+
+fileStore_path = "./collective"
```

Comment:
Use ``tempfile`` module to generate temp dir. And delete the temp dir after each UT.